Assignment1/src/main/java/Classes/Main.java

```diff
@@ -0,0 +1,12 @@
+package Classes;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Item[] items = new Item[10];
```

Comment:
since both methods takeInputs and printData are from CalculateTax class, this field can be declared inside CalculateTax class.